cmd/did-method-rest/go.sum

```diff
@@ -219,10 +219,12 @@ github.com/syndtr/goleveldb v1.0.0/go.mod h1:ZVVdQEZoIme9iO1Ch2Jdy24qqXrMMOU6lpP
 github.com/teserakt-io/golang-ed25519 v0.0.0-20200315192543-8255be791ce4 h1:Sq/68UWgBzKT+pLTUTkSf0jS2IUwwXLFlZmeh+nAzQM=
 github.com/teserakt-io/golang-ed25519 v0.0.0-20200315192543-8255be791ce4/go.mod h1:9PdLyPiZIiW3UopXyRnPYyjUXSpiQNHRLu8fOsR3o8M=
 github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5/go.mod h1:ncp9v5uamzpCO7NfCPTXjqaC+bZgJeR0sMTm6dMHP7U=
-github.com/trustbloc/edge-core v0.1.3-0.20200414220734-842cc197e692 h1:hGMF+YSdep1hpqrzZI6S9p4a5x2poPorunzoeQXByzY=
-github.com/trustbloc/edge-core v0.1.3-0.20200414220734-842cc197e692/go.mod h1:mmgEfSIoNjshnVdxEtPwh2CLs0EaBaaejgI2KNsOEOE=
+github.com/trustbloc/edge-core v0.1.3 h1:AE9XMBIVR5p67PE7ywkY4X6ILXT0gAXeIBGfOwMCcVc=
+github.com/trustbloc/edge-core v0.1.3/go.mod h1:mmgEfSIoNjshnVdxEtPwh2CLs0EaBaaejgI2KNsOEOE=
 github.com/trustbloc/sidetree-core-go v0.1.3-0.20200430203822-5e12db11f149 h1:V1rrqI38qtR8vYQvE4MwNT1GrmkdDigsyP8fMfkxTB4=
 github.com/trustbloc/sidetree-core-go v0.1.3-0.20200430203822-5e12db11f149/go.mod h1:xCuMVdRtXiCghr58Dcd1RW9t0lCDXPPjkSiACzKGaZc=
```

Comment:
Sidetree core should be release

Reply:
@Moopli I guess, you need to run go mod tidy inside cmd/did-method-rest folder.